### DISCUSSION TITLE
how to display a long string text for node?

### DISCUSSION THREAD

**Discussion by furynerd** [mermaid-js/mermaid#5794] [Q&A] [answered]
hi, im new to js and mmd.

here is a html for a study:
```html
<html>

<body>
    mmd view
    <div id="mmd" class="mermaid">
        graph LR
        a-->b[12345678901234567890123]
        c-->d[123456789012345678901234]
    </div>
    <script src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js"></script>
    <script>mermaid.initialize();</script>
</body>

</html>
```
it is found the text label for node `d` cannot diplayed (just empty), i think it is some config related (text length exceeds config). 

![图片](https://github.com/user-attachments/assets/058950ec-d16f-44e1-97e1-b8aa971bf1a7)

how can i improve my html code to solve this?

thanks very much

**Comment by aloisklink** [comment, marked as answer]
Hi @furyNerd, this is a known bug with the Firefox web browser, see https://github.com/mermaid-js/mermaid/issues/5785 and https://github.com/mermaid-js/mermaid/issues/5787

As a temporary work-around, you can set `config.flowchart.htmlLabels: false` which works around this issue, e.g.

```mermaid-example
---
config:
    flowchart:
        htmlLabels: false
---
graph LR
    a-->b[12345678901234567890123]
    c-->d[123456789012345678901234]
```

See https://mermaid.live/edit#pako:eNp1jrkOwjAQRH8l2jqWuA8XVJSpoMQUi72JI_mIHFsIRfl3DElDwVSzqzfSG0B6RcCBMSac9K5uGy5ckVMb_5QaQ5zvT3S0psIHmZ4XNZqehPsOm4CdLqrLRCJjp8dtuVpvtrv94bj4afeJkZlR_5jNHUqwFCy2KrsNn4mAqMmSAJ6rohqTiQKEGzOKKfrry0ngMSQqIfjUaOBfwxJSpzDSucVsaefv-AYzDkqD

Alternatively, you could try an older version of Mermaid, e.g. `https://cdn.jsdelivr.net/npm/mermaid@10.9.1/dist/mermaid.min.js`, where that bug doesn't exist.

**Reply by furynerd** [reply]
it works with `@10.9.1` and yes im using firefox, thanks a lot
